src/styles/_global-nav.scss

```diff
@@ -604,6 +604,11 @@ a.nav-skip-button {
   display: none;
 }
 
+@media (min-width: $screen-sm) {
+  .global-nav--component .global-nav--row-desktop {
+    height: 68.5px;
+  }
+}
```

Comment:
testing solution to [another bug](https://circleci.atlassian.net/browse/DD-329?atlOrigin=eyJpIjoiMTFiODE5MGNkZTQ4NDJmZWFkY2UzYzY2Y2VhYTEyNjMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ), will be removed from this PR before merging.

Reply:
Are you still planing on removing this?

Reply:
done!